test/functional/api/v2/media_controller_test.rb

```diff
@@ -26,7 +26,7 @@ class Api::V2::MediaControllerTest < ActionController::TestCase
     assert_difference('Medium.count', +1) do
       post :create, { :medium => new_medium }
     end
-    assert_response :created
+    assert_response :success
```

Comment:
Hm, I think this change is an indication of a bug - we should keep the 201 Created response for POST/create actions.  Is that possible?